dashboards-plugins/.meta

```diff
@@ -1,14 +1,16 @@
 {
   "projects": {
-    "queryWorkbenchDashboards": "git@github.com:opensearch-project/sql.git",
+    "queryWorkbenchDashboards": "git@github.com:opensearch-project/dashboards-query-workbench.git",
     "alertingDashboards": "git@github.com:opensearch-project/alerting-dashboards-plugin.git",
-    "notificationsDashboards": "git@github.com:opensearch-project/notifications.git",
+    "notificationsDashboards": "git@github.com:opensearch-project/dashboards-notifications.git",
     "securityDashboards": "git@github.com:opensearch-project/security-dashboards-plugin",
     "indexManagementDashboards": "git@github.com:opensearch-project/index-management-dashboards-plugin.git",
     "anomalyDetectionDashboards": "git@github.com:opensearch-project/anomaly-detection-dashboards-plugin.git",
     "reportsDashboards": "git@github.com:opensearch-project/dashboards-reports.git",
     "observabilityDashboards": "git@github.com:opensearch-project/observability.git",
     "ganttChartDashboards": "git@github.com:opensearch-project/dashboards-visualizations.git",
-    "customImportMapDashboards": "git@github.com:opensearch-project/dashboards-maps.git"
+    "customImportMapDashboards": "git@github.com:opensearch-project/dashboards-maps.git",
+    "searchRelevanceDashboards": "git@github.com:opensearch-project/dashboards-search-relevance.git",
+    "opensearch-dashboards-functional-test": "git@github.com:opensearch-project/opensearch-dashboards-functional-test.git"
```

Comment:
This isn't really a dashboards plugin, so I wasn't sure what to use for the key here.

Should this one be moved to the top-level .meta file?

Reply:
This meta file is only for plugins, I would rather remove repo's which dont release plugins.

Reply:
Is there a better place to put `opensearch-dashboards-functional-test`?

I saw on https://github.com/opensearch-project/opensearch-build/issues/3081 that the automation to cut per-repo release issues initially missed it and @gaiksaya manually cut the issue a week later. I'd like to do what I can to make sure that every repo gets hit by the automation next time. (I see that all of these repos are present in https://github.com/opensearch-project/project-meta/blob/main/.meta )

Reply:
The project-meta contains repositories that are not a part of distribution release cycle. Rather are stand-alone components. I can add `opensearch-dashboards-functional-test` under dashboards as in a way that fits. Let me know what you think.

Reply:
As long as you guys are convinced opening issues across dashboards plugins is relevant to `opensearch-dashboards-functional-test`, I am good with it.